src/test/java/de/tum/in/www1/artemis/lecture/LectureIntegrationTest.java

```diff
@@ -178,14 +184,17 @@ void updateLecture_correctRequestBody_shouldUpdateLecture() throws Exception {
         Lecture originalLecture = lectureRepository.findById(lecture1.getId()).get();
         originalLecture.setTitle("Updated");
         originalLecture.setDescription("Updated");
-        String uniqueChannelName = "test" + UUID.randomUUID().toString().substring(0, 8);
-        originalLecture.setChannelName(uniqueChannelName);
+        String channelName = "test" + UUID.randomUUID().toString().substring(0, 8);
```

Comment:
We should try to avoid random data in tests when it is not needed since randomness can lead to flakiness. It seems like this has gone unnoticed in a previous PR but I'd appreciate if you could fix this ^^

Reply:
I removed the randomness from this line and another one within this test file.